src/app/screens/Home/index.tsx

```diff
@@ -248,7 +248,7 @@ function Home() {
 
                   if (lnData[0].method === "lnurl") {
                     const lnurl = lnData[0].address;
-                    const lnurlDetails = await lnurlLib.getDetails(lnurl); // throws if invalid.
+                    const lnurlDetails = await lnurlLib.getDetails(lnurl);
```

Comment:
it stills throws if it's invalid. 
does that comment hurt?

Reply:
Before we didn't even handle the error case. Now that we do that I think it's clearer what to expect. Yes, it can still throw but we don't need to wrote this as an extra comment I guess. Can re-add it though.